pkg/controllers/user/cis/clusterScanHandler.go

```diff
@@ -380,6 +389,29 @@ func (csh *cisScanHandler) deployApp(appInfo *appInfo) error {
 	return nil
 }
 
+// collectTaints collect all taints on kubernetes nodes except node.kubernetes.io/*
+func (csh *cisScanHandler) collectTaints() (map[string]string, error) {
+	r := map[string]string{}
+	selector := labels.NewSelector()
+	nodes, err := csh.nodeLister.List("", selector)
```

Comment:
Here we are using Lister to list all nodes; however we are relying on nodelister to be updated when node goes down - so there could be some delay for lister cache getting updated?
What if CIS scan gets scheduled when lister is not yet updated? KB will get launched to the nonactive node and report generation will get stuck. Can the user cancel the CIS scan in this case?

Reply:
Confirmed with @sowmyav27 and @StrongMonkey that a user can delete the current running scan if it gets stuck in the reporting state because of the above timing.